src/inference/src/os/win/win_system_conf.cpp

```diff
@@ -17,33 +17,26 @@
 
 namespace ov {
 
-struct CPU {
-    int _processors = 0;
-    int _sockets = 0;
-    int _cores = 0;
-
-    std::vector<std::vector<int>> _proc_type_table;
-    std::vector<std::vector<int>> _cpu_mapping_table;
-
-    CPU() {
-        DWORD len = 0;
-        if (GetLogicalProcessorInformationEx(RelationAll, nullptr, &len) ||
-            GetLastError() != ERROR_INSUFFICIENT_BUFFER) {
-            return;
-        }
+void init_cpu(int& _processors,
+              int& _sockets,
+              int& _cores,
+              std::vector<std::vector<int>>& _proc_type_table,
+              std::vector<std::vector<int>>& _cpu_mapping_table) {
```

Comment:
should we create a struct which describes current CPU and return it here?
5 values passed as a reference look strange here

Reply:
Ok, done.